src/libmeasurement_kit/collector/collector.cpp

```diff
@@ -291,6 +291,8 @@ mk_collector_close_response_t *mk_collector_close(
 
 struct mk_collector_resubmit_request {
   std::string content;
+  std::string software_name = "libmeasurement_kit";
```

Comment:
It's probably worth noting somewhere that the default software_name changed from `measurement-kit` to `libmeasurement_kit`. I don't see there being any problems with that, but it's probably useful to document it at least in the changelog.

Reply:
I use `libmeasurement_kit` to indicate the library and `measurement-kit` to indicate the binary executable. Here the difference is subtle and is basically telling anyone checking the logs etc that we didn't change the default. It's basically a way to distinguish between the library and the executable that by default is using the library.

Reply:
(Ah, and I used `measurement-kit` in the previous diff as a placeholder name.)